cmd/unused/internal/ui/interactive.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func Interactive(ctx context.Context, options Options) error {
-	m := interactive.New(options.Providers, options.ExtraColumns, options.Filter.Key, options.Filter.Value)
+	m := interactive.New(options.Providers, options.ExtraColumns, options.Filter.Key, options.Filter.Value, options.DryRun)
```

Comment:
nit: should we just pass `options` for the `New()` call some time in the future?
wdyt?

Reply:
IIRC I tried doing that in the past but there was a circular dependency and it wasn't possible. I will add a note for it, tho, as it will definitely make things more clear.

Reply:
Yup, correct, I just confirmed that doing this adds a cyclic dependency. I suppose files could be moved and it would be possible, but that's definitely out of the scope of this PR.